beets/util/artresizer.py

```diff
@@ -503,13 +600,17 @@ def reformat(self, path_in, new_format, deinterlaced=True):
 
         fname, ext = os.path.splitext(path_in)
         path_new = fname + b'.' + new_format.encode('utf8')
-        func = BACKEND_CONVERT_IMAGE_FORMAT[self.method[0]]
 
         # allows the exception to propagate, while still making sure a changed
         # file path was removed
         result_path = path_in
         try:
-            result_path = func(self, path_in, path_new, deinterlaced)
+            result_path = self.local_method.convert_format(
+                path_in, path_new, deinterlaced
+            )
+        except Exception:
+            # FIXME: Should probably issue a warning?
+            pass
```

Comment:
I could be missing something, but I think this method didn't previously suppress exceptions—the `finally` clause does cleanup even in the case of exceptions, but the exception was still propagated.

Reply:
You're right, the `finally` must have confused me — I've restored the old (and more sensible) behaviour now.